tests/Gaufrette/Functional/Adapter/AzureMultiContainerBlobStorageTest.php

```diff
@@ -0,0 +1,178 @@
+<?php
+
+namespace Gaufrette\Functional\Adapter;
+
+/**
+ * Class AzureMultiContainerBlobStorageTest
+ * @group AzureBlobStorage
+ * @group AzureMultiContainerBlobStorage
+ */
+class AzureMultiContainerBlobStorageTest extends FunctionalTestCase
```

Comment:
I don't understand this one. I've added a new test suite, haven't disabled the existing one. By default, both will be run (and I have run both test sets).